scripts/validation/validate-examples.js

```diff
@@ -0,0 +1,35 @@
+const { execSync } = require('child_process');
+const glob = require('glob');
+
+// Use glob to find all YAML files in the examples directory
+const files = glob.sync('./examples/**/*.{yml,yaml}');
```

Comment:
I believe there are some files you won't be able to validate since they are not AsyncAPI documents such as the files located in https://github.com/asyncapi/spec/tree/master/examples/social-media/common. Not sure if there are more, you will need to check.

Reply:
Yes, it is like that. 
You can see the output:

![image](https://github.com/asyncapi/spec/assets/99868037/4491cef6-8870-4035-b43f-416e73d91095)

So, what can be done about it? Exclude these? Or something else?

Reply:
Exclude them, yes.

Reply:
> Exclude them, yes.

Got it!